### PR TITLE
Implement band math endpoint

### PR DESCRIPTION
**Pull request by dionhaefner**

### CODE REVIEW

**Comment by dionhaefner** [comment]
@mrpgraae @j08lue @nmheim Could I ask one of you to take over from here? The feature is done from my perspective, but there are still some architectural decisions to be made (#20), the handler and flask route need test coverage, and the endpoint has to be documented.

File: terracotta/server/flask_api.py [hunk 1]
```diff
@@ -110,6 +110,10 @@ def create_app(debug: bool = False, profile: bool = False) -> Flask:
 
     if profile:
         from werkzeug.contrib.profiler import ProfilerMiddleware
-        new_app.wsgi_app = ProfilerMiddleware(new_app.wsgi_app, restrictions=[30])
+        setattr(
+            new_app,
+            'wsgi_app',
+            ProfilerMiddleware(new_app.wsgi_app, restrictions=[30]
+        )
```
Comment (j08lue) [inline terracotta/server/flask_api.py]: ??

Comment (mrpgraae) [reply terracotta/server/flask_api.py]: I guess because this also works if `new_app.wsgi_app` doesn't exist already.

Comment (mrpgraae) [reply terracotta/server/flask_api.py]: No acutally, it won't. So I would also like to know 😄

Comment (dionhaefner) [reply terracotta/server/flask_api.py]: It's the recommended workaround for https://github.com/python/mypy/issues/2427

**Comment by dionhaefner** [comment]
Done from my side. Needs more documentation, but so do all endpoints.

**Pull request merged**